stl/inc/xutility

```diff
@@ -4559,11 +4567,6 @@ _BidIt2 _Copy_backward_memmove(_BidIt1 _First, _BidIt1 _Last, _BidIt2 _Dest) {
     return static_cast<_BidIt2>(_CSTD memmove(_Dest_ch - _Count, _First_ch, _Count));
 }
 
-template <class _BidIt1, class _BidIt2>

```

Comment:
This and its forward friend are the only things I could imagine to break something